### PR TITLE
Address comments from Eric V

### PR DESCRIPTION
**Pull request by tfpauly**
Addresses #93 and #87

### CODE REVIEW

File: draft-ietf-taps-transport-security.md [hunk 1]
```diff
@@ -134,16 +134,19 @@ are both included despite overlapping features. As such, this survey is not limi
...
-options specified in {{?RFC2385}}.) One prime use case of TCP-AO is for protecting BGP connections.
-Similarly, AH adds per-datagram authenticity and adds similar replay protection. Despite these
-improvements, neither protocol sees general use and both lack critical properties important for emergent
-transport security protocols: confidentiality, privacy protections, and agility. Such protocols are thus
-omitted from this survey.
+Authentication-only protocols such as TCP-AO {{?RFC5925}} and IPsec Authentication Header (AH) {{?RFC4302}}
+are excluded from this survey. TCP-AO adds authentication to long-lived TCP connections, e.g., replay
+protection  with per-packet Message Authentication Codes. (TCP-AO obsoletes TCP MD5 "signature"
+options specified in {{?RFC2385}}.) One primary use case of TCP-AO is for protecting BGP connections.
+Similarly, AH adds per-datagram authenticationa and integrity, along with replay protection. Despite
```
Comment (renghardt) [inline draft-ietf-taps-transport-security.md]: nit: s/authenticationa/authentication/

**Review by renghardt — COMMENTED** [review]
Looks good so far, modulo one typo.

I was wondering if we want to include "integrity protection" to our definition of Security Protocol, too, as Eric suggested. But that sentence says "such as", so perhaps it's fine if we don't enumerate all possible features.

Also, Eric suggested we change our generic use of the term "record protocol", as our definition in Section 2 clashes with it.
Maybe we want to change the definition in Section 2 instead to match our use and remove the "Record" definition? I don't think we actually use "Record" as "Framed protocol messages" in the rest of the document, do we?

File: draft-ietf-taps-transport-security.md (hunk 1 above)
Comment (chris-wood) [inline draft-ietf-taps-transport-security.md]:
```suggestion
Similarly, AH adds per-datagram authentication and integrity, along with replay protection. Despite
```

File: draft-ietf-taps-transport-security.md [hunk 2]
```diff
@@ -134,16 +134,19 @@ are both included despite overlapping features. As such, this survey is not limi
...
-from this survey. TCP-AO adds authenticity protections to long-lived TCP connections, e.g., replay
-protection  with per-packet Message Authentication Codes. (This protocol obsoletes TCP MD5 "signature"
-options specified in {{?RFC2385}}.) One prime use case of TCP-AO is for protecting BGP connections.
-Similarly, AH adds per-datagram authenticity and adds similar replay protection. Despite these
-improvements, neither protocol sees general use and both lack critical properties important for emergent
-transport security protocols: confidentiality, privacy protections, and agility. Such protocols are thus
-omitted from this survey.
+Authentication-only protocols such as TCP-AO {{?RFC5925}} and IPsec Authentication Header (AH) {{?RFC4302}}
+are excluded from this survey. TCP-AO adds authentication to long-lived TCP connections, e.g., replay
+protection  with per-packet Message Authentication Codes. (TCP-AO obsoletes TCP MD5 "signature"
```
Comment (chris-wood) [inline draft-ietf-taps-transport-security.md]:
```suggestion
protection with per-packet Message Authentication Codes. (TCP-AO obsoletes TCP MD5 "signature"
```

File: draft-ietf-taps-transport-security.md [hunk 3]
```diff
@@ -134,16 +134,19 @@ are both included despite overlapping features. As such, this survey is not limi
...
+options specified in {{?RFC2385}}.) One primary use case of TCP-AO is for protecting BGP connections.
+Similarly, AH adds per-datagram authentication and integrity, along with replay protection. Despite
+these improvements, neither protocol sees general use and both lack critical properties important for emergent transport
+security protocols, such as confidentiality and privacy protections. Such protocols are thus omitted from this survey.
+
+The protocols surveyed in this document are point-to-point protocols. Security of multicast protocols
+is not covered.
```
Comment (chris-wood) [inline draft-ietf-taps-transport-security.md]:
```suggestion
This document only surveys point-to-point protocols; multicast protocols are out of scope.
```

**Review by chris-wood — APPROVED** [review]
@tfpauly LGTM with a couple nits!

**Pull request merged**